pkg/devfile/sample_test.go

```diff
@@ -72,7 +72,7 @@ func TestGetRegistrySamples(t *testing.T) {
 					DisplayName: "Basic Python",
 					Description: "A simple Hello World application using Python",
 					Tags:        []string{"Python"},
-					Icon:        "https://www.python.org/static/community_logos/python-logo-generic.svg",
+					Icon:        "https://raw.githubusercontent.com/devfile-samples/devfile-stack-icons/main/python.svg",
```

Comment:
Should we consider relaxing some of these tests so they don't break when things change? For instance, checking if python exists, has some icon, etc, but not necessarily all of the exact values.

Let's merge this to unblock CI, but maybe as a follow up.

Reply:
OK I see this is already tracked by https://issues.redhat.com/browse/OCPBUGS-1678, thanks 👍

Reply:
ccing @jerolimov